swift/extractor/BUILD.bazel

```diff
@@ -1,5 +1,10 @@
 cc_binary(
     name = "extractor",
     srcs = ["main.cpp"],
+    target_compatible_with = select({
```

Comment:
Irritating that we need the select here, instead of just listing linux and macos 😬

Reply:
yeah, I also got surprised the list was an AND and not an OR, they should have given a way to do general logic expressions here...